tests/test_trusted-execution.yml

```
@@ -0,0 +1,132 @@
+---
+- name: Basic sanity test for fapolicyd
+  hosts: localhost
```

```suggestion
  hosts: all
```
the test framework will provide the correct hostname

---

DONE.